### ISSUE TITLE
Ex00 Part 4

### ISSUE THREAD

**Issue by naveedbinrais** [NorhanKhaled/SkillsAcademy-CAD#20]
I installed IBM Cloud CLI but when I check the version, I get the following error:

C:\Program Files\IBM\Cloud\bin>ibmcloud --version
FAILED
Configuration error: open I:\.bluemix\plugins\config.json: The system cannot find the path specified.

How to set the correct path?

**Comment by NorhanKhaled** [comment]
are you sure you run the command that install the ibmcloud ?

**Comment by naveedbinrais** [comment]
Yes. I run the .exe file and it installed IBM CLI on my machine in the path. C:\Program Files\IBM\Cloud\bin
After that, I went to the bin directory and tried to test the version of ibmcloud and got this error.
